File: Tugas coding modul 4/coding c/PRAK404-2310817120005-Noviana_Nur_Aisyah.c
#include <stdio.h>

int main() {
    int pilihan;
    float angka1, angka2, hasil;

    while (1) {
        printf("Pilih program:\n1. Penjumlahan\n2. Pengurangan\n3. Perkalian\n4. Pembagian\n5. Exit\n");
        printf("Masukkan pilihan: ");
        scanf("%d", &pilihan);

        if (pilihan == 1) {
            printf("Masukkan nilai pertama: ");
            scanf("%f", &angka1);
            printf("Masukkan nilai kedua: ");
            scanf("%f", &angka2);
            hasil = angka1 + angka2;
            printf("Hasil Penjumlahan antara %.2f dengan %.2f adalah %.2f\n\n", angka1, angka2, hasil);
        } else if (pilihan == 2) {
            printf("Masukkan nilai pertama: ");
            scanf("%f", &angka1);
            printf("Masukkan nilai kedua: ");
            scanf("%f", &angka2);
            hasil = angka1 - angka2;
            printf("Hasil pengurangan antara %.2f dengan %.2f adalah %.2f\n\n", angka1, angka2, hasil);
        } else if (pilihan == 3) {
            printf("Masukkan nilai pertama: ");
            scanf("%f", &angka1);
            printf("Masukkan nilai kedua: ");
            scanf("%f", &angka2);
            hasil = angka1 * angka2;
            printf("Hasil perkalian antara %.2f dengan %.2f adalah %.2f\n\n", angka1, angka2, hasil);
        } else if (pilihan == 4) {
            printf("Masukkan nilai pertama: ");
            scanf("%f", &angka1);
            printf("Masukkan nilai kedua: ");
            scanf("%f", &angka2);
            hasil = angka1 / angka2;
            printf("Hasil pembagian antara %.2f dengan %.2f adalah %.2f\n\n", angka1, angka2, hasil);
        } else if (pilihan == 5) {
            printf("Terimakasih, telah menggunakan kalkulator, Noviana Nur Aisyah");
            break;
        } else {
            printf("Input anda salah, silahkan coba lagi\n\n");
        }
    }

    return 0;
}

File: Tugas coding modul 4/coding py/PRAK404-2310817120005-Noviana_Nur_Aisyah.py
while True:
        print("Pilih program:\n1. Penjumlahan\n2. Pengurangan\n3. Perkalian\n4. Pembagian\n5. Exit")
        pilihan = int(input("Masukkan pilihan: "))

        if pilihan == 1:
            angka1 = float(input("Masukkan nilai pertama: "))
            angka2 = float(input("Masukkan nilai kedua: "))
            hasil = angka1 + angka2
            print(f"Hasil Penjumlahan antara {angka1:.2f} dengan {angka2:.2f} adalah {hasil:.2f}\n")

        elif pilihan == 2:
            angka1 = float(input("Masukkan nilai pertama: "))
            angka2 = float(input("Masukkan nilai kedua: "))
            hasil = angka1 - angka2
            print(f"Hasil pengurangan antara {angka1:.2f} dengan {angka2:.2f} adalah {hasil:.2f}\n")

        elif pilihan == 3:
            angka1 = float(input("Masukkan nilai pertama: "))
            angka2 = float(input("Masukkan nilai kedua: "))
            hasil = angka1 * angka2
            print(f"Hasil perkalian antara {angka1:.2f} dengan {angka2:.2f} adalah {hasil:.2f}\n")

        elif pilihan == 4:
            angka1 = float(input("Masukkan nilai pertama: "))
            angka2 = float(input("Masukkan nilai kedua: "))
            hasil = angka1 / angka2
            print(f"Hasil pembagian antara {angka1:.2f} dengan {angka2:.2f} adalah {hasil:.2f}\n")

        elif pilihan == 5:
            print("Terimakasih, telah menggunakan kalkulator, Noviana Nur Aisyah")
            break

        else:
            print("Input anda salah, silahkan coba lagi\n")